packages/slate-react/src/utils/environment.ts

```diff
@@ -14,3 +14,7 @@ export const IS_FIREFOX =
 export const IS_SAFARI =
   typeof navigator !== 'undefined' &&
   /Version\/[\d\.]+.*Safari/.test(navigator.userAgent)
+
```

Comment:
whats the userAgent for modern edge?

Reply:
It's "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.74 Safari/537.36 Edg/79.0.309.43"

According to https://docs.microsoft.com/en-us/microsoft-edge/web-platform/user-agent-string

Reply:
```suggestion
// "modern" edge was released at 79.x
```